src/main/java/edu/uc/inventorymanager/dto/Item.java

```diff
@@ -1,12 +1,15 @@
 package edu.uc.inventorymanager.dto;
 
 import lombok.Data;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
 
+@RequiredArgsConstructor
```

Comment:
@RequiredArgsConstructor creates a constructor besides the default one with no parameters